pages/bare_metal_cloud/dedicated_servers/bring-your-own-linux/guide.en-gb.md

```diff
@@ -0,0 +1,167 @@
+- Access to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=we) (for the method "[Deployment via Control Panel](#viacontrolpanel)")
+- Access to the [OVHcloud API](/pages/manage_and_operate/api/first-steps) (for the section "[Deployment via API](#viaapi)" of this guide)
+- Your image must be smaller than the Server RAM minus 3GiB
+
+> [!warning]
+>
+> A new installation with BYOLinux will erase all the data on the server.
+>
+
+## Instructions
+
+**Technical limitations**
+
+There are some technical limitations linked to the use of physical products such as dedicated servers. Here is a non-exhaustive list, to keep in mind during your deployment preparation:
+
+- Boot type: **uefi** or **legacy**
+- Partition type: **MBR** or **GPT**
+- Image format: **qcow2**
+
+**Deployment methods**
+
+- [Deployment via the Control Panel](#viacontrolpanel): allows you to simply deploy your image using the OVHcloud Control Panel.
+- [Deployment via API](#viaapi): you can use the OVHcloud API to integrate images into your own scripts to automate deployments.
+
+### Deploy your image via the Control Panel <a name="viapanel"></a>
+
+Log in to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB) and go to the `Bare Metal Cloud`{.action} section, then select your server under `Dedicated servers`{.action}.
+
+In the `General information` tab, click on the `...`{.action} button next to "System (OS)" and then on `Install`{.action}.
+
+![BringYourOwnLinux Control Panel 01](images/byolinux-controlpanel01.png){.thumbnail}
+
+In the window that appears, select `Install from OVHcloud template`{.action} and click on `Next`{.action}.
+
+![BringYourOwnLinux Control Panel 02](images/byolinux-controlpanel02.png){.thumbnail}
+
+In the windoe that appears, select `CUSTOM`, in the menu, then `Bring Your Own Linux - byolinux`, and click on `Next`{.action}.
+
+![BringYourOwnLinux Control Panel 03](images/byolinux-controlpanel03.png){.thumbnail}
+
+You will be redirected to the configuration page. Make sure your image URL is in the correct format. Complete the rest of the required fields on this page. Once you have confirmed that the information is correct, click `Confirm`{.action}.
+
+You can find more details on the options in the "[Deployment options](#options)" section below.
+
+For more information and examples about Cloud-Init's ConfigDrive, please consult the official documentation on [this page](https://cloudinit.readthedocs.io/en/22.1_a/topics/examples.html).
+
+![BringYourOwnLinux Control Panel 04](images/byolinux-controlpanel04.png){.thumbnail}
+
+### Deploy your image via the APIs <a name="viaapi"></a>
+
+> [!api]
+>
+> @api {v1} /dedicated/server/{serviceName}/install/start POST /dedicated/server/{serviceName}/install/start
+>
+
+The base payload for the `install/start` is 
+
+```json
```

Comment:
remove this please, it will change very soon several time, I don't want/need to update this